src/main/java/kr/codesquad/secondhand/application/chat/ChatRoomService.java

```diff
@@ -54,4 +60,14 @@ public Long createChatRoom(Long itemId, Long senderId) {
         item.increaseChatCount();
         return chatRoom.getId();
     }
+
+    public boolean isUpToDate(Long messageIndex) {
+        if (messageIndex == null) {
+            return false;
+        }
+        if (chatLogRepository.existsByIdGreaterThan(messageIndex)) {
+            return false;
+        }
+        return true;
```

Comment:
여기서는 이렇게 할 수 있을 것 같아요!
```suggestion
        return chatLogRepository.existsByIdGreaterThan(messageIndex) == false;
```